src/main/java/com/rapid7/client/dcerpc/io/PacketInput.java

```diff
@@ -68,6 +68,19 @@ public int readReferentID()
         return result;
     }
 
+    public byte[] readRawBytes(int length)
```

Comment:
Is this method needed? I think readFully provides the same functionality.

`
byte[] data = new byte[length]
packetInput.readFully(data)
`
Then the data can be passed to whatever that needs it.

Reply:
@bli-r7 ,  yep,  it provides  with readFully method but for the convenience  we need to wrapper class (some places it frequently needs as below case) 
 i.e. byte[] sidIdentifierAuthority = packetIn.readRawBytes(6);

Reply:
it another use cases from  `https://github.com/rapid7/smbj-rpc/compare/master...skhare-r7:lookupSids#diff-a8a5526085ae6feb39e39ec4f7ac05d0R58`